mep/local_settings.py.sample

```diff
@@ -26,14 +26,24 @@ DATABASES = {
         'ENGINE': 'django.db.backends.sqlite3',
         'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
```

Comment:
Ah, good catch. We probably don't really want anyone using sqlite for dev (although I occasionally switch to it for checking things or generating fixtures). Perhaps we should make the msyql the default and/or remove the sqlite example entirely.

Reply:
fwiw I support making mysql the default also. i changed cdhweb's local settings sample to use postgres, since we're switching to postgres there.

Reply:
I'll swap them! Let me know if you feel strongly about deleting sqlite